definitions/npm/object-hash_v1.x.x/flow_v0.25.x-/object-hash_v1.x.x.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Flow libdef for 'object-hash'
+ * See https://www.npmjs.com/package/object-hash
+ * by Vincent Driessen, 2018-12-21
+ */
+
+// @flow strict
+
+type $npm$objectHash$Options = {|
+  algorithm?: "sha1" | "sha256" | "md5",
+  excludeValues?: boolean,
+  encoding?: "buffer" | "hex" | "binary" | "base64",
+  ignoreUnknown?: boolean,
+  unorderedArrays?: boolean,
+  unorderedSets?: boolean,
+  unorderedObjects?: boolean,
+  excludeKeys?: string => boolean
+|};
+
+declare module "object-hash" {
+  declare module.exports: (
```

Comment:
same as in https://github.com/flow-typed/flow-typed/pull/3047#discussion_r245523167, you can make the types better scoped using `declare export`

Reply:
Done in 4fb2587.